tests/test_cli.py

```diff
@@ -46,6 +48,8 @@ def assert_expected_file(self, result, output_path):
 
     def test_schema_convert_cli(self, runner, helpers):
         data_model_csv_path = helpers.get_data_path("example.model.csv")
+        data_model_pickle_path = helpers.get_data_path("example.model.pickle")
```

Comment:
For these I would try to find a way to supply these parameters via parmeterize, you can send the models via one parameter and the output type parameter via another. 
Then you could call the appropriate runner, based on your current parameters, do some checks to see where you care and call the appropriate asserts. Does that make sense?

Reply:
yeah, this makes sense. I'll play around with it!